### PR TITLE
Mobile App Versions

### PR DESCRIPTION
**Pull request by nicholaskillin**
This PR will add tabs to the warnings app. The first new tab will list out all of our mobile apps and the current version for each of those apps.

### CODE REVIEW

**Comment by dougbevis** [comment]
I can upload my changes whenever, but I need to be added as a contributor before GitHub will let me push.

**Comment by nicholaskillin** [comment]
@dougbevis I'm not able to add you Reagan will have to do that. Just wanted to mention though that while I've added some tabs at the top they aren't functional yet, so this isn't quite ready for you to take it over. Once I have them working I'll be sure to let you know.

**Comment by dougbevis** [comment]
@nicholaskillin I have the tabs functioning with JS in my code. I had some free time last night, so I threw it together. Once I upload, feel free to adjust styling or give feedback!

**Comment by nicholaskillin** [comment]
@dougbevis Sounds good. Once @ReaganHelms adds you as a contributor I'll take a look at your changes.

File: warning_v2/assets/iframe.html [hunk 1]
```diff
@@ -6,19 +6,103 @@
 </head>
 <body>
 
-  <div id="requester-content"></div>
-  <div id="ticket-content"></div>
-  <div id="notes-content"></div>
+<!-- nav --->
```
[the remark this answers is not in the document]
Comment (dougbevis) [reply warning_v2/assets/iframe.html]: See Commit Message on new push below.

File: warning_v2/assets/iframe.html [hunk 2]
```diff
@@ -120,6 +204,93 @@
...
+          check();
+          cca();
+          head();
+          music();
+          people();
+          resources();
+
+
+
+      </script>
```
[the remark this answers is not in the document]
Comment (dougbevis) [reply warning_v2/assets/iframe.html]: See Commit message on new push.

**Comment by nicholaskillin** [comment]
This looks great @dougbevis I made a few tweaks, rebasing with master and then cleaning up the formatting a bit. Go ahead and take another look and if it all looks good to you then I think we are safe to merge to master.

File: warning_v2/assets/main.css [hunk 3]
```diff
@@ -77,3 +78,54 @@
...
+  	display: inline-block; 
+  	text-align: center; 
+  	padding-bottom: 10px;
+}
+  .logo {
+
+}
+  .version {
+
+}
```
Comment (nicholaskillin) [inline warning_v2/assets/main.css:103]: @dougbevis Just noticed these. If we aren't going to do anything with the class in the CSS it's probably a good idea to go ahead and remove these for now. If we need them later we can add them in then.

**Comment by dougbevis** [comment]
This looks good to me! I'll let @ReaganHelms handle the merge to master.

**Pull request merged**